config/langs.toml

```diff
@@ -688,19 +688,22 @@ version = '8.2'
 website = 'https://www.vim.org'
 example = '''
 " Printing
-execute "normal! iHello, World!\r"
+i
+Hello, World!
+.
 
 " Looping
-for i in range(0, 9)
-  execute "normal! i".i."\r"
-endfor
+for i in range(10)
+pu=i
```

Comment:
can we indent the `pu` for clarity? Ideally 4 spaces to match the other langs. I know people will remove whitespace when golfing but the examples should be uber readable imo.